docs/src/languages/rust.md

```diff
@@ -3,7 +3,42 @@
 - Tree Sitter: [tree-sitter-rust](https://github.com/tree-sitter/tree-sitter-rust)
 - Language Server: [rust-analyzer](https://github.com/rust-lang/rust-analyzer)
 
-### Target directory
+## Inlay Hints
+
+Zed sets the following initialization options for inlay hints:
```

Comment:
Is this the default, or must this be configured by the user?

Reply:
It's not, actually, just a configuration example.
There's no extra hint settings Zed sets for r-a.